src/models/Movie/parseMovieData.ts

```diff
@@ -0,0 +1,28 @@
+import { MovieModel } from './type';
+import { GenreContentfulEntry } from '../Genre/GenreModel';
+
+// TODO: Fix contentful types, take out the any
+export default function parseMovieData(entry: any): MovieModel {
```

Comment:
I know this is ugly, but I want to create a follow-up ticket to generate the Contentful types correctly. I tried 2 libraries and they weren't updated to support content@10